### DISCUSSION TITLE
Show where a tap actually happened when watching a replay

### DISCUSSION THREAD

**Discussion by memesnightmares** [ppy/osu#15181] [Ideas]
When watching my replays, I'm frequently unsure of how I missed a note, did I mis-aim, did I tap outside of the timing window, etc. It would be nice to have a visual indication of where all of my taps were when watching a replay so I could see this sort of information. I also think it would just look cool, maybe a dot on screen at every tap that fades out after however many ms when watching replays.

**Comment by integerrr** [comment]
#5681 #14720 #13154

**Reply by memesnightmares** [reply]
Those are similar, and I'm excited about their potential, but this request is different. This is for live indications of where each click happens as you watch a replay. So, when there is a tap/click in a replay, a dot would appear under my cursor so that I can see where exactly I tapped, not in the progress bar or break time screens.

**Reply by integerrr** [reply]
![image](https://user-images.githubusercontent.com/7279624/137780671-eb32647b-a2a8-4e32-b153-d2a86b0d701a.png)

**Reply by memesnightmares** [reply]
Yes, both of those issues/discussions are about indicators in the progress bar, not in the playfield. This feature request is about the playfield. I'm aware that feature requests like this are low priority, but I wanted to make it so that I didn't forget and so that it would be visible if there's ever room for things like this on the roadmap.

That screencap is about the difference between showing misses in the progress bar during gameplay vs during replay viewing which are the same thing. The playfield is different.

**Reply by integerrr** [reply]
#13154

**Reply by memesnightmares** [reply]
Ah, see that feature request sounds similar (although a little less specific/actionable). This is loosely a duplicate of that request. Thank you for the link, I've upvoted it :P
